pom.xml

```diff
@@ -501,11 +504,37 @@
         </plugins>
       </build>
 
+
+      <dependencyManagement>
```

Comment:
add scope `test` to all dependencies.

Reply:
Added the scope test to all dependencies.